app-office/texmaker/texmaker-5.0.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A nice LaTeX-IDE"
+DESCRIPTION="Texmaker is a free, modern and cross-platform LaTeX editor for \
+linux, macosx, haiku and windows systems that integrates many tools needed to \
+develop documents with LaTeX, in just one application.
+Texmaker includes unicode support, spell checking, auto-completion, code \
+folding and a built-in pdf viewer with synctex support and continuous view \
+mode.
+Texmaker is easy to use and to configure."
+HOMEPAGE="https://xm1math.net/texmaker/"
+COPYRIGHT="2003-2019 Pascal Brachet"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://xm1math.net/texmaker/texmaker-$portVersion.tar.bz2"
+CHECKSUM_SHA256="6074dffa82c431f8727c39e046f62cb70f5fe1457f158b46306e8f490b4f326f"
+PATCHES="texmaker-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Comment:
If this is tested to work on x86, then you can remove the untested mark for only x86.

```suggestion
ARCHITECTURES="!x86_gcc2 x86 x86_64"
```

Reply:
This has been done for a while because we have no x86 primary architecture anymore, do we still plan on using that or keeping it for historical reason?